app/frontend/controllers/app_controller.js

```diff
@@ -250,61 +259,65 @@ class AppController {
   initGlobalShortCuts() {
     let shortCut = 'ctrl+c';
     if (platformHelper.isMac()) {
-      shortCut = 'command+c';
+        shortCut = 'command+c';
     }
 
     Mousetrap.bind(shortCut, async () => {
-      await this.verse_selection.copySelectedVerseTextToClipboard();
-      return false;
+        if (this.dictionary_controller.isSelectionActive()) {
+            await this.dictionary_controller.copySelectedTextToClipboard();
```

Comment:
The function `copySelectedTextToClipboard()` is not async and therefore does not need to be called with `await`.

Reply:
Done. Removed await keyword from verse_selection.copySelectedVerseTextToClipboard() call